### DISCUSSION TITLE
@storybook/types TypescriptOptions not includes checkOptions

### DISCUSSION THREAD

**Discussion by SuperRun** [storybookjs/storybook#25102] [Help]
### Summary

@storybook/types `TypescriptOptions` not includes `checkOptions` ![image](https://github.com/storybookjs/storybook/assets/28804901/0f679e9c-0688-43d7-980d-d0bb01daa577)
But the api doc shows the `checkOptions` I wonder if the checkOptions will work
https://storybook.js.org/docs/api/main-config-typescript

**Comment by kylegach** [comment]
Yes, `checkOptions` will work. Can you point me to that source file you took a screenshot of, please?
